backend/app/controllers/spree/admin/orders_controller.rb

```diff
@@ -34,7 +34,7 @@ def index
           params[:q][:completed_at_lt] = params[:q].delete(:created_at_lt)
         end
 
-        @search = Order.preload(:user).accessible_by(current_ability, :index).ransack(params[:q])
+        @search = Order.preload(:user).accessible_by(current_ability, :index).ransack(params[:q].to_unsafe_h)
```

Comment:
@mauazua The ransack whitelisting attrs is already part of model doing it again didn't seem right to me. It accepts query param hash and if nothing is whitelisted in that hash it will itself raise exception.